Makefile

```diff
@@ -1,9 +1,3 @@
-# Update this variable, then run 'make setver'
-# Naming convention:
-#	for stable releases we use "1.0.0" format
-#   for pre-releases, we use   "1.0.0-beta.2" format
-VERSION=1.2.5
-
```

Comment:
Why did we take this out?  We should be able to override on the Make command line if necessary.